### PR TITLE
Feature/v7

### PR DESCRIPTION
**Pull request by mikaelmetthey**
## Description
Updated content & navigation
Updated routes.js

## Checklist

- [ ] Tested against our [testing policy](https://github.com/nhsuk/nhsuk-frontend/blob/master/docs/contributing/testing.md) (Resolution, Browser & Accessibility)
- [ ] CHANGELOG entry

### CODE REVIEW

File: app/views/v7-Welsh/overview.html [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+      <h2>Beth fydd cofrestru i ymchwilwyr gysylltu â chi yn galw amdano</h2>
+      <p>Gallwch chi gofrestru ar-lein. Byddwn ni’n gofyn rhai cwestiynau ichi ynglŷn â’ch hun, ac yna’n gofyn am eich caniatâd i ymchwilwyr ar yr astudiaethau brechlyn gysylltu â chi.</p>
+      <p>Os byddwch chi’n cofrestru, cedwir eich manylion yn ddiogel. Caiff y rhain eu rhannu ag ymchwilwyr sy’n meddwl y gallech chi fod yn addas ar gyfer astudiaeth y maen nhw’n gweithio arni yn unig. Bydd yr ymchwilwyr yna’n cysylltu â chi i ddweud mwy am yr astudiaeth, ac i ateb unrhyw gwestiynau sydd gennych chi.</p>
+      <p>Os byddwch chi’n cofrestru i ymchwilwyr gysylltu â chi, nid oes yn rhaid ichi gymryd rhan mewn astudiaeth na siarad ag ymchwilwyr. Eich dewis chi ydy hyn.</p>
+      <p>Gallwch chi dynnu’ch caniatâd yn ôl ar unrhyw adeg.</p>
+
+      {{ actionLink({
+        "text": "Cofrestru i ymchwilwyr gysylltu â chi ar gyfer astudiaethau brechlyn coronafeirws",
+        "href": "/v5-Welsh/"
```
Comment (mcheung-nhs) [inline app/views/v7-Welsh/overview.html:27]: Should this be` /v7-Welsh/` ?

File: app/views/v7-Welsh/overview.html [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+            https://assets.nhs.uk/prod/images/nihr-logo2.width-767.png 767w,
+            https://assets.nhs.uk/prod/images/nihr-logo2.width-1019.png 1019w,
+            https://assets.nhs.uk/prod/images/nihr-logo2.width-1125.png 1125w,
+            https://assets.nhs.uk/prod/images/nihr-logo2.width-1534.png 1534w,
+          " alt="Logo image for the National Institute for Health Research">
+      </figure>
+
+      {{ actionLink({
+        "text": "Cofrestru i ymchwilwyr gysylltu â chi ar gyfer astudiaethau brechlyn coronafeirws",
+        "href": "/v5-Welsh/"
```
Comment (mcheung-nhs) [inline app/views/v7-Welsh/overview.html:179]: Should this be` /v7-Welsh/` ?

**Review by mcheung-nhs — CHANGES_REQUESTED** [review]

**Comment by mikaelmetthey** [comment]
Well spotted, thanks Michael.
Updated to /v7-Welsh/

File: app/views/v7-Welsh/email-loader.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+  </div>
+{% endblock %}
+
+{% block pageScripts %}
+  <script>
+    $(document).ready(function () {
+        // Handler for .ready() called.
+        window.setTimeout(function () {
+            location.href = "/v5-Welsh/email-sent";
```
Comment (mcheung-nhs) [inline app/views/v7-Welsh/email-loader.html:27]: Found another - `/v7-Welsh-email-sent`

File: app/views/v7-Welsh/email-sent-busy.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      <div class="nhsuk-inset-text nhsuk-u-margin-top-5 nhsuk-u-margin-bottom-5">
+        <span class="nhsuk-u-visually-hidden">Pwysig</span>
+        <p>Mae yna lawer o bobl yn cofrestru ar hyn o bryd. Mae’n bosibl y bydd angen i chi aros yn hirach nag arfer am yr e-bost. </p>
+      </div>
+
+      <p>Os ydy’r cyfeiriad e-bost uchod yn anghywir, gallwch chi <a href='/{{ version }}/email'>newid eich cyfeiriad e-bost</a></p>
+
+      <p>Ni allwch chi gofrestru i ni gysylltu â chi ar gyfer astudiaethau brechlyn coronafeirws nes eich bod wedi cadarnhau eich cyfeiriad e-bost. </p>
+
+      <p class="nhsuk-body-s nhsuk-u-margin-top-9"><a href="/emails-v3-Welsh/confirm-email">Next step (prototype only)</a></p>
```
Comment (mcheung-nhs) [inline app/views/v7-Welsh/email-sent-busy.html:31]: Should this be `/emails-v4-Welsh/confirm-email` ?

File: app/views/v7-Welsh/email-sent.html [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        <p>Ni allwch chi gofrestru i ni gysylltu â chi ar gyfer astudiaethau brechlyn coronafeirws nes eich bod wedi cadarnhau eich cyfeiriad e-bost.</p>
+      </div>
+
+      <p>Os ydy’r cyfeiriad e-bost uchod yn anghywir, gallwch chi <a href='/{{ version }}/email'>newid eich cyfeiriad e-bost</a></p>
+
+      <h2 class="nhsuk-heading-s nhsuk-u-padding-top-2">Os na fyddwch chi’n cael e-bost</h2>
+      <p>Gwiriwch eich ffolder sbam neu sothach.</p>
+      <p>Gadewch ychydig o funudau i’r e-bost gyrraedd. Os oes yna lawer o bobl yn cofrestru ar yr un pryd, mae’n bosibl y bydd yn rhaid i chi aros ychydig yn hirach nag arfer am yr e-bost.</p>
+
+      <p class="nhsuk-body-s nhsuk-u-margin-top-9"><a href="/emails-v3-Welsh/confirm-email">Next step (prototype only)</a></p>
```
Comment (mcheung-nhs) [inline app/views/v7-Welsh/email-sent.html:32]: Should this be `/emails-v4-Welsh/confirm-email` ?

**Review by mcheung-nhs — CHANGES_REQUESTED** [review]

File: app/views/v7-Welsh/confirmation.html [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+      <p>Os gwnaethoch gytuno i NHS Digital gysylltu â chi ynglŷn â sut mae'r astudiaethau brechlyn coronafeirws yn mynd, a datblygiadau i'r gwasanaeth hwn gan gynnwys y cyfle i gymryd rhan mewn ymchwil iechyd arall, byddant yn cysylltu â chi pan fydd gwybodaeth newydd i'w rhannu. Byddwch yn gallu dad-danysgrifio unrhyw amser.</p>
+
+      <div class="nhsuk-inset-text nhsuk-u-margin-top-5 nhsuk-u-margin-bottom-5">
+        <p>Darganfyddwch yn y polisi preifatrwydd <a href="/privacy">sut y bydd eich data yn cael ei brosesu gan ddefnyddio'r gwasanaeth hwn</a>.</p>
+      </div>
+
+      <h2>Tynnu eich caniatâd yn ôl</h2>
+      <p>Darganfyddwch sut i <a href="/opt-out-v2-Welsh/confirm-email">dynnu'ch caniatâd yn ôl i ni gysylltu â chi ar gyfer astudiaethau brechlyn coronafeirws</a>.</p>
+
+      <p class="nhsuk-body-s nhsuk-u-margin-top-9"><a href="/emails-v3-Welsh/confirmation">See confirmation email (prototype only)</a></p>
```
Comment (mcheung-nhs) [inline app/views/v7-Welsh/confirmation.html:48]: Should this be `/emails-v4-Welsh/confirmation` ?

**Review by mcheung-nhs — CHANGES_REQUESTED** [review]

**Comment by mikaelmetthey** [comment]
Updated the Welsh v7

**Review by mcheung-nhs — APPROVED** [review]
All good!

**Pull request merged**